src/core/plugins/json-schema-2020-12-samples/fn/main.js

```diff
@@ -21,6 +21,9 @@ export const sampleFromSchemaGeneric = (
   exampleOverride = undefined,
   respectXML = false
 ) => {
+  // there is nothing to generate schema from
+  if (schema == null && exampleOverride === undefined) return undefined
```

Comment:
Could `exampleOverride` ever be `null` here as well? As in, if `null` somehow gets passed to this function?

Reply:
It can. `null` is a valid JavaScript example.